autoload/ale/organize_imports.vim

```diff
@@ -17,7 +17,9 @@ function! ale#organize_imports#HandleTSServerResponse(conn_id, response) abort
     \       'description': 'Organize Imports',
     \       'changes': l:file_code_edits,
     \   },
-    \   {}
+    \   {
+    \       'conn_id': a:conn_id,
+    \   },
```

Comment:
Add "should_save" in the options here so the files are saved and we get no errors when "nohidden" is set:

```
{
    'conn_id':  a:conn_id,
    'should_save': 1,
}
```

Reply:
Done, also added an option to disable this behavior.